docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -646,6 +646,108 @@
     "tab.mode(dropna=False)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b248fef1",
+   "metadata": {},
+   "source": [
+    "### Table.sem()\n",
+    "\n",
+    "```\n",
+    "Table.sem(axis=0, skipna=True, numeric_only=False, ddof=0)\n",
+    "```\n",
+    "Return unbiased standard error of the mean over requested axis. Normalized by N-1 by default. This can be changed using the ddof argument\n",
+    "\n",
+    "**Parameters:**\n",
+    "\n",
+    "| Name         | Type | Description                                                                      | Default |\n",
+    "| :----------: | :--: | :------------------------------------------------------------------------------- | :-----: |\n",
+    "| axis         | int  | The axis to calculate the sum across 0 is columns, 1 is rows.                    | 0       |\n",
+    "| skipna       | bool | not yet implemented                                           | True    |\n",
+    "| numeric_only | bool | Only use columns of the table that are of a numeric data type.                   | False   |\n",
+    "| ddof    | int  | Delta Degrees of Freedom. The divisor used in calculations is N - ddof, where N represents the number of elements. | 1 |\n",
+    "\n",
+    "**Returns:**\n",
+    "\n",
+    "| Type               | Description                                                          |\n",
+    "| :----------------: | :------------------------------------------------------------------- |\n",
+    "| Table         | The std across each row / column with the key corresponding to the row number or column name. |"
```

Comment:
Change both Type and Description. We are returning a Dictionary (`pykx.wrappers.Dictionary`) and we are not calculating the std.

Reply:
👍